### DISCUSSION TITLE
Techdraw workbench

### DISCUSSION THREAD

**Discussion by Isaina** [FreeCAD/FreeCAD#9943] [Code contribution discussions]
Hi

can someone  add 2D pipes hvac Line and multi Line elements with Raiser and autodetect point to annotate Bim drawings in 2d like 

https://alcad.b-cdn.net/DEMO-2D-MEP.mp4

**Comment by Roy-043** [comment]
This is not material for a Code contributing discussion. Please start a discussion in the FreeCAD forum instead: https://forum.freecad.org/viewforum.php?f=8.

Closing this.
